src/Raven.Server/Documents/Replication/IncomingReplicationHandler.cs

```diff
@@ -1139,7 +1139,7 @@ protected override long ExecuteCmd(DocumentsOperationContext context)
                             case AttachmentReplicationItem attachment:
 
                                 var localAttachment = database.DocumentsStorage.AttachmentsStorage.GetAttachmentByKey(context, attachment.Key);
-                                if (_replicationInfo.ReplicatedAttachmentStreams.TryGetValue(attachment.Base64Hash, out var attachmentStream))
+                                if (_replicationInfo.ReplicatedAttachmentStreams != null && _replicationInfo.ReplicatedAttachmentStreams.TryGetValue(attachment.Base64Hash, out var attachmentStream))
```

Comment:
When `replicationInfo.ReplicatedAttachmentStreams` can be `null` now?

Reply:
I think only for the `TestingStuff` I added